### PR TITLE
Update testnet-validator-setup.md

### PR DESCRIPTION
**Pull request by napelvs**
- My [recent PR](https://github.com/persistenceOne/persistence-docs/pull/12) has moved the node creation to the relevant file, leaving this file only for instructions on how to create a validator and other useful commands.
- I've removed the 'Before Genesis' Section due to its redundancy.
- I've updated the links to the 'Explorer', and 'Version'.

### CODE REVIEW

File: build/validators/testnet-validator-setup.md [hunk 1]
```diff
@@ -1,156 +1,168 @@
...
+  --chain-id=test-core-1 \
+  --gas="auto" \
+  --gas-prices="0.0025uxprt" \
+  --from=<key_name> \
+  --commission-rate="0.10"
+```
 
-### Before genesis: NOW CLOSED
+{% hint style="warning" %}
+Please note that some parameters such as `commission-max-rate` and `commission-max-change-rate` cannot be changed once your validator is up and running.
```
Comment (ajeet97) [inline build/validators/testnet-validator-setup.md:116]: `{% hint %}` does not work in github markdown 🤔 

You can use these for special messages
```
> **Note**
> This is a note

> **Warning**
> This is a warning
```
Preview
> **Note**
> This is a note

> **Warning**
> This is a warning

Comment (napelvs) [reply build/validators/testnet-validator-setup.md]: Thanks for pointing this out. I'll use it going forward.

Change implemented.

File: build/validators/testnet-validator-setup.md [hunk 2]
```diff
@@ -1,156 +1,168 @@
...
+
+	You should now see your validator in [one of the Persistence explorers](https://testnet.ping.pub/test-core-1/uptime).
+7. **Congratulations!** You've just created a Persistence Testnet Validator.
+
+### View Validator Description
+
+View the validator's information with this command:
+
+```bash
+persistenceCore query staking validator <validator_address>
```
[the remark this answers is not in the document]
Comment (napelvs) [reply build/validators/testnet-validator-setup.md]: Great command! I'll use it going forward.

Change implemented.

**Review by ajeet97 — APPROVED** [review]
Change the gas price in the example command as well.
Else LGTM 👍

**Pull request merged**